components/app/myrw/widgets/tabs/list/helpers.js

```diff
@@ -0,0 +1,25 @@
+export const getQueryParams = (state = {}, props) => {
+  const {
+    sort,
+    pagination,
+    search
+  } = state;
+  const {
+    user: { id },
+    subtab
+  } = props;
+  const { page, limit } = pagination;
+  const isCollection = !['my_widgets', 'favourites'].includes(subtab);
+
+  return ({
+    'page[size]': limit,
+    'page[number]': page,
+    sort: sort === 'asc' ? 'updatedAt' : '-updatedAt',
+    userId: id,
+    ...search && search.length && { name: search },
```

Comment:
@andresgnlez Just a quick question, why using the spread operator here instead of the variable `search` directly ?

Reply:
It's a way to conditionally add properties to the object. I check previously if the search value is right, otherwise, we would always be sending search empty to the API (which is fine, it doesn't affect the results, but I think this is more cleaner).

Reply:
Oh I see now. Thanks for the clarification 👍